app/src/main/java/com/webview/arun/helloworldapp/LoginActivity.java

```diff
@@ -350,38 +350,10 @@ protected void onCancelled() {
     }
 
     public void EmptyMethod() { }
-
-    public void EM3() { }
-    
-    public void EM4() { }
-    
-    public void EM5() {
-           int i =0;
-        int j=0;
-    }
-    
-    public void EM6() {
-           int i =0;
-        int j=0;
-    }
-    public void EM7() {
-           int i =0;
-        int j=0;
+    public void N1() {
+           int ii =0;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "ii" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)